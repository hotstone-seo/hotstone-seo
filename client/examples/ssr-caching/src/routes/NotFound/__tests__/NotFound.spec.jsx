import React from 'react';
import NotFound from '../';

test('Should render NotFound correctly', () => {
  const tree = mount(<NotFound />);
  
  expect(tree).toMatchSnapshot();
});
