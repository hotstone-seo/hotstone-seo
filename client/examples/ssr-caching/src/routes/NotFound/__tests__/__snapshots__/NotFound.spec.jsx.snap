// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFound correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Router
      history={
        Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "?order_id=39982221&order_hash=2a71d91259eefafd4bea3465050d4fa7ecb4186e",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <NotFound>
        <Grid
          bsClass="container"
          componentClass="div"
          fluid={false}
        >
          <div
            className="container"
          >
            <Row
              bsClass="row"
              componentClass="div"
              style={
                Object {
                  "marginTop": 80,
                }
              }
            >
              <div
                className="row"
                style={
                  Object {
                    "marginTop": 80,
                  }
                }
              >
                <Col
                  bsClass="col"
                  className="text-center"
                  componentClass="div"
                  xs={12}
                >
                  <div
                    className="text-center col-xs-12"
                  >
                    <h1>
                      Ooopss... Are you lost ?
                    </h1>
                    <div
                      style={
                        Object {
                          "marginTop": 20,
                        }
                      }
                    >
                      <Button
                        active={false}
                        block={false}
                        bsClass="btn"
                        bsStyle="primary"
                        disabled={false}
                        onClick={[Function]}
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          type="button"
                        >
                          Go back
                        </button>
                      </Button>
                    </div>
                  </div>
                </Col>
              </div>
            </Row>
          </div>
        </Grid>
      </NotFound>
    </Router>
  </BrowserRouter>
</Provider>
`;
